src/main/java/nl/tudelft/ewi/devhub/server/database/AssignTAs.java

```diff
@@ -0,0 +1,54 @@
+package nl.tudelft.ewi.devhub.server.database;
```

Comment:
Note that we actually do this and it is a requirement for this year. Manual reassignment is possible, but not desirable. CC @mjwsteenbergen as he is the original author of the assignment script we are using atm.